docs/releases/5.0.txt

```diff
@@ -272,6 +272,11 @@ Forms
 * The new ``assume_scheme`` argument for :class:`~django.forms.URLField` allows
   specifying a default URL scheme.
 
+* To improve accessibility and enable screen readers to associate form fields
+  with their help text, the form field now includes a new HTML attribute
+  ``aria-describedby``, which points to the corresponding help text HTML
+  element.
```

Comment:
`aria-describedby` is not a new HTML attribute. Also, the last sentence sounds redundant:
```suggestion
* In order to improve accessibility and enable screen readers to associate form
  fields with their help text, the form field now includes ``aria-describedby``
  HTML attribute.
```